src/date-range-picker/time-offset.ts

```diff
@@ -39,11 +39,12 @@ export function shiftTimeOffset(
 
   /*
     This regex matches an ISO date-time with
+    - optionally matches date-only format
```

Comment:
date-only format is still considered a valid ISO 8601 representation. It represents a calendar date without any time information. This format will be used when only the date is relevant, and the time doesn't need to be specified.

Reply:
This is a functional change, but the PR is called "Enhancing date-range-picker pages" which does not match it.

1. Do you want to extract it in another PR and annotate properly and add relevant tests?
2. Is there any other reason to bundle these two changes together?

Reply:
Removed these changes as it is no longer needed after [this change](https://github.com/cloudscape-design/components/blob/main/src/date-range-picker/utils.tsx#L82) was added in a [recent PR](https://github.com/cloudscape-design/components/pull/3111/files?w=1 )